src/pages/Events.jsx

```diff
@@ -0,0 +1,25 @@
+import { Flex, Text } from "@sparrowengg/twigs-react";
+
+const Events = () => {
+  return (
+    <Flex
+      css={{
+        width: "1080px",
```

Comment:
@9xVibee why is this comment is resolved and do you really need a fixed width???

Reply:
As i mentioned in the previous comment reply its just because Events and Advance Quarry dont have the content so temporarily i have given the widths to them!